src/PeerConnectionManager.cpp

```diff
@@ -833,14 +833,29 @@ rtc::scoped_refptr<webrtc::AudioTrackInterface> PeerConnectionManager::CreateAud
 	if (!audioSource) {
 		RTC_LOG(LS_ERROR) << "Cannot create capturer audio:" << audiourl;
 	} else {
-		std::string label = audiourl + "_audio";
-		label.erase(std::remove_if(label.begin(), label.end(), ignoreInLabel), label.end());
-		audio_track = peer_connection_factory_->CreateAudioTrack(label, audioSource);
```

Comment:
Do you like to remove audio_track creation ?

Reply:
Oh, shoot.. Need to add that missing line back:

		audio_track = peer_connection_factory_->CreateAudioTrack(label, audioSource);
Sorry!